src/api/xxAMIDOxx.xxSTACKSxx.CQRS.UnitTests/EventsTests.cs

```diff
@@ -43,13 +44,13 @@ public void EventsShouldHaveUniqueIds()
 
         public void EventNameShouldMatchOperationName()
         {
-            var definitions = typeof(MenuCreatedEvent).Assembly.GetImplementationsOf(typeof(IApplicationEvent));
+            var definitions = typeof(MenuCreated).Assembly.GetImplementationsOf(typeof(IApplicationEvent));
```

Comment:
We have created a example package with MenuCreatedEvent.
Where are we raising MenuCreated? We shouldn't be doing it. we should raise MenuCreatedEvent that exists in the package (here: https://github.com/amido/stacks-dotnet-packages-cqrs-events)

Reply:
Modified the unit test to check for the correct event name implementation